style.css

```diff
@@ -0,0 +1,718 @@
+  line-height: 62px;
+  text-align: center;
+  margin-left: 6%;
+}
+
+.nav-dropdown-item-link a :hover {
+  color: red;
+}
+
+.nav-dropdown {
+  list-style: none;
+  margin-top: 15%;
+}
+
+.mobile-hide {
+  display: none;
+}
+
+.blur-intro-section {
+  filter: blur(4px);
+  -webkit-filter: blur(4px);
+}
+
+.hidden-dropdown-menu {
+  display: none;
+}
+
+.featured-program-item-desktop {
+  display: none;
+}
+
+.program-item-mobile {
+  background-color: rgba(88, 89, 90, 0.8);
+  justify-content: center;
+  flex-direction: column;
+  text-align: center;
+  margin-right: 0.5%;
+  width: 15%;
+  padding: 1%;
+}
+
+.intro-section {
+  background-image: url("./images/background-image.png");
+  height: 100vh;
+}
+
+.introduction {
+  padding-top: 20%;
+  color: #ec5242;
+}
+
+.introduction-title {
+  font-size: 40px;
+  font-weight: 700;
+  width: 100%;
+  font-family: "Cocogoose", sans-serif;
+  transition: all 0.25s ease-in-out;
+}
+
+.introduction-title:hover {
+  color: transparent;
+  background-image: url("./images/title-background.webp");
+  background-position: center;
+  background-size: cover;
+  background-repeat: no-repeat;
+  background-clip: content-box;
+  -webkit-background-clip: text;
+}
+
+.introduction-slogan {
+  font-family: "lato", sans-serif;
+}
+
+.introduction-description {
+  width: 100%;
+  background-color: #e7e7e7;
+  border: #fff solid 2px;
+  color: #58595a;
+  padding: 0.8rem;
+  font-family: "lato", sans-serif;
+}
+
+.introduction-offers {
+  color: #272a31;
+  font-weight: 700;
+  font-size: 27px;
+}
+
+.introduction-time {
+  color: #58595a;
+  font-weight: 500;
+}
+
+.container-1 {
+  background-image: url("https://clauzzen.files.wordpress.com/2011/01/vintage-pattern_black.png");
+  background-repeat: no-repeat;
+  background-size: cover;
+  display: flex;
+  flex-direction: column;
+  padding-top: 20px;
+  padding-bottom: 20px;
+  width: auto;
+}
+
+.program-item-holder {
+  margin: 10% 3% 0 3%;
+  list-style: none;
+  padding: 0;
+}
+
+.item {
+  display: flex;
+  align-items: center;
+  background-color: #444242;
+  justify-content: space-between;
+  gap: 2%;
+  color: #d3d3d3;
+  margin-bottom: 10px;
+  margin-left: 10px;
+  list-style: none;
+  padding: 5% 5%;
+  opacity: 0.8;
+  margin-right: 10px;
+  transition: outline 0.15s ease-in-out;
+}
+
+.program-description {
+  color: #fff;
+  font-family: "Lato", sans-serif;
+  font-size: 12px;
+  letter-spacing: 1px;
+  flex-shrink: 1;
+}
+
+.item:hover {
+  outline: 2px solid #d3d3d3;
+}
+
+.item-icon {
+  padding-right: 4%;
+  margin: 15px 0;
+}
+
+.main-program {
+  font-family: "Cocogoose", sans-serif;
+  color: #d3d3d3;
+  display: flex;
+  justify-content: center;
+  width: 100%;
+}
+
+.program-title-container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.program-subtitle {
+  font-size: 1rem;
+  font-family: "Cocogoose", sans-serif;
+  color: #ec5242;
+}
+
+.hr-title {
+  border: 1px solid #ec5242;
+  width: 20%;
+}
```

Comment:
- [ ] I think it will be better if we follow the original design as much as possible as parsing a given design is a crucial skill for a front-end dev, currently you see the underline after the section heading does not have the correct width as per the design :point_down: 
![image](https://user-images.githubusercontent.com/97521557/217486820-7e3b3dc3-5665-4cee-97f1-52df774cc9b0.png)

how it should be :point_down: 
![image](https://user-images.githubusercontent.com/97521557/217486932-ab2bd38b-4a3c-4ad9-a97f-f21e471933e7.png)

Fix this for all section headings :+1: